charts/spark-operator-chart/README.md

```diff
@@ -66,10 +66,16 @@ The command removes all the Kubernetes components associated with the chart and
 | metrics.enable | bool | `true` | Enable prometheus mertic scraping |
 | metrics.endpoint | string | `"/metrics"` | Metrics serving endpoint |
 | metrics.port | int | `10254` | Metrics port |
+| metrics.portName | string | `metrics` | Metrics port name |
 | metrics.prefix | string | `""` | Metric prefix, will be added to all exported metrics |
 | nameOverride | string | `""` | String to partially override `spark-operator.fullname` template (will maintain the release name) |
 | nodeSelector | object | `{}` | Node labels for pod assignment |
 | podAnnotations | object | `{}` | Additional annotations to add to the pod |
+| podMonitor.create | bool| `false` | Create a prometheus pod monitor for operator's pod. Note that prometheus metrics should be enabled as well.|
```

Comment:
Should we use `enable` instead of `create`?

Reply:
Updated.